README.md

```diff
@@ -58,14 +58,24 @@ pip install kili
 
 ![](./recipes/img/api_key.gif)
 
-- In your favourite IDE :
+- Add the `KILI_API_KEY` variable in your bash environment (or in the settings of your favorite IDE):
+```bash
+export KILI_API_KEY='MY API KEY'
```

Comment:
```suggestion
export KILI_USER_API_KEY='MY API KEY'
```
According to the code, this seems to be the proper variable used in kili/authentication.py no ?

Reply:
it is overridden when it is called:
https://github.com/kili-technology/kili-playground/blob/e7430bb8a79cc4932b5f68e671598ae9dc81b274/kili/client.py#L80